plasmapy/physics/distribution.py

```diff
@@ -0,0 +1,85 @@
+"""Functions to deal with distribution : generate, fit, calculate"""
+
+
+from astropy import units
+
+from astropy.units import (UnitConversionError, UnitsError, quantity_input,
+                           Quantity)
+
+from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, e)
+from ..atomic import (ion_mass, charge_state)
+
+import numpy as np
+
+from ..utils import _check_quantity, check_relativistic, check_quantity
+
+
+def Maxwellian_1D(v,T,part ="e", V_drift = 0* units.m/units.s):
```

Comment:
Seems like a nice place for [@astropy.units.quantity_input](http://docs.astropy.org/en/stable/api/astropy.units.quantity_input.html#astropy.units.quantity_input) :D

Reply:
good point !